### PR TITLE
Add legal link to footer

### LINKED ISSUE #50: Add legal link to footer
> We checked with the IT team, here's the response: We must keep everything according to legal, however, Contact Us and Volvo Shop can go to another area.

- We should add [Do Not Sell or Share My Personal Information](https://www.volvogroup.com/en/tools/privacy/data-subject-rights.html) back to the footer
- The link to switch region (select market) will be put in the footer, as it’s being removed from the header (https://github.com/Netcentric/vg-volvotrucks-us-rd/issues/3)

- [Designs](https://www.figma.com/file/IcqreRKnNYyDVMc28yoOeh/Homepage?type=design&node-id=4152-20617&mode=design&t=62XnRXSWmLicTFq5-0) provided here
- Rename the v2-footer to footer

### PR DESCRIPTION
**Pull request by cogniSyb**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #50

Test URLs:
- Before: https://main--vg-volvotrucks-us-rd--netcentric.hlx.page/
- After: https://50-add-legal-link-to-footer--vg-volvotrucks-us-rd--netcentric.hlx.page/

### CODE REVIEW

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -101,38 +129,35 @@ function openExternalLinksInNewTab(footer) {
...
-  });
-  footer.querySelectorAll('a[href^="https://www.youtube.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-youtube"></i> ${anchor.innerHTML}`;
-  });
-  footer.querySelectorAll('a[href^="https://www.instagram.com/"]').forEach((anchor) => {
-    anchor.innerHTML = `<i class="fa fa-instagram"></i> ${anchor.innerHTML}`;
-  });
+function findList(h3Element) {
+  let nextElement = h3Element.nextElementSibling;
+  while (nextElement && !nextElement.classList.contains('footer-list-item')) {
```
Comment (kesiah) [inline blocks/footer/footer.js:134]: `const list = h3Element.parentNode.querySelector('.footer-list-item')` would be better? 🤔

File: icons/globe.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...ill="#141414" fill-rule="evenodd" d="M8.32 3.6a9.02 9.02 0 0 0-3.98 2.98h2.23c.2-.55.44-1.06.7-1.53.31-.54.66-1.03 1.05-1.45ZM3.68 7.58a8.95 8.95 0 0 0-1.16 3.93h3.12c.04-1.4.25-2.74.6-3.93H3.67ZM2.52 12.5a8.95 8.95 0 0 0 1.15 3.93h2.56c-.34-1.2-.55-2.53-.59-3.93H2.52Zm1.81 4.93a9.02 9.02 0 0 0 4 3 7.97 7.97 0 0 1-1.06-1.47c-.26-.47-.5-.98-.7-1.53H4.33Zm10.37 3a9.02 9.02 0 0 0 3.99-3h-2.23c-.21.55-.45 1.06-.71 1.53-.31.54-.66 1.04-1.05 1.46Zm4.64-4a8.95 8.95 0 0 0 1.16-3.93h-3.12c-.04 1.4-.25 2.74-.6 3.93h2.56Zm1.15-4.93a8.95 8.95 0 0 0-1.15-3.93H16.8c.34 1.2.55 2.53.59 3.93h3.11Zm-1.8-4.93A9.02 9.02 0 0 0 14.7 3.6c.39.42.74.91 1.05 1.45.26.47.5.98.7 1.53h2.23ZM11.5 2a10 10 0 1 0 0 20 10 10 0 0 0 0-20Zm3.37 3.54c.18.32.35.66.5 1.03H12v-3.5c1.05.2 2.06 1.05 2.87 2.47ZM12 7.58h3.73c.37 1.17.6 2.5.64 3.93h-4.37V7.58Zm0 4.93h4.37a14.74 14.74 0 0 1-.64 3.93h-3.73v-3.93Zm0 4.93h3.37c-.15.37-.32.71-.5 1.03-.8 1.42-1.82 2.27-2.87 2.48v-3.5Zm-1 0v3.51c-1.05-.2-2.06-1.06-2.87-2.48a9.6 9.6 0 0 1-.5-1.03h3.37Zm0-1H7.28c-.37-1.17-.6-2.5-.64-3.93h4.37v3.93Zm0-4.93H6.64c.04-1.43.27-2.76.64-3.93H11v3.93Zm0-4.93H7.64c.16-.37.32-.71.5-1.03.8-1.42 1.82-2.28 2.87-2.48v3.5Z" clip-rule="evenodd"/></svg>
```
Comment (kesiah) [inline icons/globe.svg:1]: Can we update the `fill` value by currentColor?

**Review by kesiah — CHANGES_REQUESTED** [review]
The footer tests seem to fail :(

**Comment by kesiah** [comment]
> > The footer tests seem to fail :(
> 
> @kesiah I checked all the v2-footer files, there were no tests present. Due to renaming files the old test is now triggered. I’ve seen the test, but I don’t agree with the setup as it is dependent on content. Shall we discuss this in a call?

@cogniSyb we are not testing anything apart from common.js file. The rest is old code. For me, we can get rid of it.

**Review by kesiah — APPROVED** [review]

**Review by markovukiceviccn — APPROVED** [review]

**Pull request merged**